docs/visualizing-your-sql-queries/plotting-with-plotly.md

```diff
@@ -1,14 +1,409 @@
 ---
-jupyter:
-  jupytext:
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.14.5
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
 ---
 
 # Plotting with `plotly`
 
+Plotly is a visualization library specifically designed for dynamic interactive plots. The library offers several additional effects to visualizations, such as zooming, panning, and hovering effects. This library is especially known for being easily deployed with web applications.
 
+For more on plotly, visit: https://plotly.com/python/
 
+Let's see how we can apply plotly to our familiar bank marketing data sets.
+
+## Install - execute this once.
+
+```{important}
+<b>Note:</b> The `--save` and `%sqlcmd` features used require the latest JupySQL version. Ensure you run the code below to update JupySQL.
+```
+
+This code installs JupySQL, DuckDB, and Pandas in your environment. We will be using these moving forward.
+
+```{code-cell} ipython3
+%pip install jupysql --upgrade duckdb-engine pandas plotly --quiet
+```
+
+## Load the data
+
+```{important}
+<b>Note:</b> If you are following these lessons locally and <b>not</b> on Google Colab, then there is no need to load the data again.
+```
+
+This section was covered in detail in the previous tutorial: [Joining Data in SQL](https://ploomber-sql.readthedocs.io/en/latest/intro-to-sql/joining-data-in-sql.html#load-the-data). We will be using the same data in this tutorial as well.
+
+```{code-cell} ipython3
+import sys
+import plotly.express as px
+
+sys.path.insert(0, "../../")
```

Comment:
Do we need this?

Reply:
I believe we do in order to access the `banking.py` script in the root folder.